src/ur5e_arm.hpp

```diff
@@ -2,46 +2,16 @@
 
 #include <ur_client_library/control/trajectory_point_interface.h>
 #include <ur_client_library/types.h>
```

Comment:
Do you still need both includes? Or can you get away with just `types.h` now?

Reply:
needed both